### ISSUE TITLE
how to display many circle in one page?

### ISSUE THREAD

**Issue by samsomlan** [zircleUI/zircleUI#32]
hi Is there anyway to display many circle in one page. I want to represent one IOT device by one circle. so I need to display a lot circle in one page

**Comment by tinchox5** [comment]
Hi @samsomlan, currently zircle can only have one instance. Also, it shows just one main circle per view. 

But to achieve what you want, you can change de border width of the main circle (z-view) and just expose the z-spots... Let me a day or so and I will show a codepen with the idea.

**Comment by samsomlan** [comment]
thanks a lot!

**Comment by tinchox5** [comment]
@samsomlan, please see this codepen:
https://codepen.io/zircle/pen/WqPGPx

Zircle is not currently designed to have multiples views, but a way to create multiple "view" containers is to hide the border of the main view (z-view) by adding this css style `<z-view style="border: none">` (codepen js line 2). Also you need to hide the dashed border `.z-outer-circle {
  border: none !important
}` and add `!important` to override the default style (surely there are many ways to achieve the same)

The last part is to increase the size property of the `z-spot` to `xxl` to simulate the same size of the `z-view` and adjust the distance property to avoid overlaps. 

To sum up, you need to hide the default `z-view` and use `z-spots` as `z-view`.

**Comment by samsomlan** [comment]
this help me a lot! thanks you
